src/agent/onefuzz-task/src/tasks/task_logger.rs

```diff
@@ -45,7 +48,7 @@ trait LogWriter<T>: Send + Sync {
 
 /// Writes logs on azure blobs
 pub struct BlobLogWriter {
-    container_client: Arc<ContainerClient>,
```

Comment:
Are there perf implications here, wrt an underlying HTTP client?

Reply:
From what I can tell the client is still shared. `ContainerClient` has a `StorageClient` has a `Pipeline` which is a `Vec<Arc<dyn Policy>>`, of which the last entry is the HTTP transport (phew!).